localization/english/wc_modifiers_l_english.yml

```diff
@@ -4,4 +4,6 @@
  holy_order_government_opinion:0 "[holy_order|E] Ruler Opinion"
  
  rebuilding_kingdom_modifier:0 "Rebuilding Kingdom"
- 
\ No newline at end of file
+
+ stasis_trap_stunned:0 "Stunned"
```

Comment:
The name of the modifier stopped showing.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/120220112-4de42600-c24d-11eb-88ac-5c62b9cc212c.png)

</details>